### DISCUSSION TITLE
Changing Modal's zIndex

### DISCUSSION THREAD

**Discussion by art-in** [chakra-ui/chakra-ui#6688] [Q&A] [answered]
I've a special modal which I want to show above everything else (including any other modals/overlays/etc).

But seems like `zIndex` prop on `ModalContent` is ignored.

```jsx
<Modal isOpen={true}>
  <ModalOverlay zIndex='2000'  /> {/* Works here */}
  <ModalContent zIndex='2000' />  {/* Ignored here */}
</Modal>
```

Is this a bug?

Or maybe I'm supposed to manage z-index of modals some other way? How would you change z-index for one particular modal then?

**Comment by joyfeel** [comment, marked as answer]
try this

```js
<Modal>
  <ModalOverlay zIndex='2000' />
  <ModalContent 
    containerProps={{
      zIndex: '2000',
    }} 
  /> 
</Modal>
```
